pkg/controller.v1beta1/experiment/experiment_controller.go

```diff
@@ -114,33 +114,19 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 // addWatch adds a new Controller to mgr with r as the reconcile.Reconciler
 func addWatch(mgr manager.Manager, c controller.Controller) error {
 	// Watch for changes to Experiment
-	err := c.Watch(&source.Kind{Type: &experimentsv1beta1.Experiment{}}, &handler.EnqueueRequestForObject{})
+	err := c.Watch(source.Kind(mgr.GetCache(), &experimentsv1beta1.Experiment{}), &handler.EnqueueRequestForObject{})
```

Comment:
The source.Kind{} typed. was replaced with source.Kind() function:

https://github.com/kubernetes-sigs/controller-runtime/blob/116a1b831fffe7ccc3c8145306c3e1a3b1b14ffa/pkg/source/source.go#L60-L63